tests/app/celery/test_provider_tasks.py

```diff
@@ -17,6 +19,7 @@
     NotificationTechnicalFailureException,
     InvalidProviderException,
 )
+from app.mobile_app import DEAFULT_MOBILE_APP_TYPE
```

Comment:
see comment above about fixing this longstanding typo `DEAFULT_MOBILE_APP_TYPE`

Reply:
Updating.